automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/PingSpec.scala

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import org.scalatest.{FreeSpec, Matchers}
+
+class PingSpec extends FreeSpec with Matchers with LeonardoTestUtils {
+  "Leonardo" - {
+    "should ping" in {
+      implicit val token = ronAuthToken
+      Leonardo.test.ping() shouldBe "OK"
```

Comment:
Are we ever going to remove the `/ping` endpoint :)

Reply:
I love it as a sanity check.  Could use a `/status` endpoint instead though